manim/plugins/import_plugins.py

```diff
@@ -0,0 +1,41 @@
+import pkg_resources
+from importlib import import_module
+from .. import config, logger
+
+__all__ = []
+
+# don't know what is better to do here!
+module_type = type(pkg_resources)
+function_type = type(import_module)
+
+plugins_requested: list = config["plugins"]
+if "" in plugins_requested:
+    plugins_requested.remove("")
+for plugin in pkg_resources.iter_entry_points("manim.plugins"):
+    if plugin.name in plugins_requested:
+        loaded_plugin = plugin.load()
+        if isinstance(loaded_plugin, module_type):
+            # it is a module so it can't be called
+            # see if __all__ is defined
+            # if it is defined use that to load all the modules necessary
+            # essentially this would be similar to `from plugin import *``
+            # if not just import the module with the plugin name
+            if hasattr(loaded_plugin, "__all__"):
+                for thing in loaded_plugin.__all__:
+                    exec(f"{thing}=loaded_plugin.{thing}")
```

Comment:
Tried searching for a cleaner way to set local variables with dynamic names, unfortunately could not find one, and `exec` seems to be the cleanest way. I'm not sure of the purpose of this review comment, I guess a general lamentation and a note to any others who might know of a cleaner way. Writing to `locals()`, `vars()`, or `globals()` is explicitly discouraged by the python docs, so that's not an option. I'm not sure if maybe preserving plugins' namespaces would make this unnecessary?